src/main/java/com/codingnomads/betty/data/api/GameInformationAPIRepository.java

```diff
@@ -20,8 +24,50 @@ public GameInformationAPIRepository(RestTemplate restTemplate) {
     @Override
     public GameInformationJSON getGameInformation(){
 
-        String url = "https://api.matchbook.com/edge/rest/events?tag-url-names=premier-league&sport-ids=15&states=open&exchange-type=back-lay&odds-type=DECIMAL&include-prices=false&price-depth=1&price-mode=aggregated&side=lay&currency=GBP&minimum-liquidity=10&include-event-participants=true&market-types=one_x_two&per-page=500";
+        String url = "https://api.matchbook.com/edge/rest/events?tag-url-names=united-states-of-america&sport-ids=15&states=open&exchange-type=back-lay&odds-type=DECIMAL&include-prices=false&price-depth=1&price-mode=aggregated&side=lay&currency=GBP&minimum-liquidity=10&include-event-participants=true&market-types=one_x_two&per-page=500";
 
         return restTemplate.getForObject(url, GameInformationJSON.class);
     }
+
+    @Override
+    public EventJSON getEventJSONforMatch(String match, GameInformationJSON gameInformationJSON) {
+
```

Comment:
Nice! No request to make changes for this PR but I do recommend you do not include a space after every line. Generally, only do it before the return statements so they are easy to read